app/templates/views/security.html

```diff
@@ -76,11 +76,6 @@ <h3 class="font-body-lg">Two-factor authentication</h3>
   </ul>
   <p>If signing in with a text message is a problem for your team, <a class="usa-link" href="{{ url_for('main.support') }}">contact us</a> to find out about using an email link instead.</p>
 
-  <img src="{{ asset_url('images/product/security-review-message.png') }}"
-    alt="Screenshot of a teat message in review with the link to 'hide personalization after sending' circled.">
```

Comment:
typo for test

Reply:
Ah, thankfully this is a deleted line!  Thanks for catching though, @terrazoon!